File: projekatWeb2/src/app/models/WorkRequest.ts
export class WorkRequest {
  id: number;
  type: string;
  status: string;
  incident: number;
  street: string;
  startedTime: Date;
  finishedTime: Date;
  userCreated: string;
  purpose: string;
  notes: string;
  urgent: boolean;
  company: string;
  phoneNumber: string;
  equipment: string;
  creationDate: string;

  constructor() {
    this.id = 0;
    this.type = '';
    this.status = 'Draft';
    this.incident = 0;
    this.street = '';
    this.startedTime = new Date();
    this.finishedTime = new Date();
    this.userCreated = '';
    this.purpose = '';
    this.notes = '';
    this.urgent = false;
    this.company = '';
    this.phoneNumber = '';
    this.equipment = '';
    this.creationDate = '';
  }
}

File: Backend/Backend/Models/WorkRequest.cs
﻿using System;
using System.Collections.Generic;

namespace Backend.Models
{
    public class WorkRequest
    {
        public int Id { get; set; }
        public string Type { get; set; }
        public string Status { get; set; }
        public int Incident { get; set; }
        public string Street { get; set; }
        public DateTime StartedTime { get; set; }
        public DateTime FinishedTime { get; set; }
        public string UserCreated { get; set; }
        public string Purpose { get; set; }
        public string Notes { get; set; }
        public bool Urgent { get; set; }
        public string Company { get; set; }
        public string PhoneNumber { get; set; }
        public DateTime CreationDate { get; set; }
        public string Equipment { get; set; }
        public List<Change> ChangeHistory { get; set; }
    }
}
